content/docs/cli/cli.md

```diff
@@ -0,0 +1,15 @@
+# CLI Usage
+
+This section describes how to use KUDO.
+
+## Setup the KUDO Kubectl Plugin
+
+How to install and setup KUDO. Details can be found [here](installation.md)
+
+## Commands
+
+An overview about all the things the KUDO CLI can do is [here](commands.md)
+
+## Examples
+
+Easy to follow examples for common use cases are [here](examples.md)
```

Comment:
Using `here` as link text is an anti-pattern, see for example https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/

```suggestion
How to [install and setup KUDO](installation.md).

## Commands

An overview about all [the things the KUDO CLI can do](commands.md).

## Examples

Easy to follow [examples for common use cases](examples.md).
```

Reply:
Another important reason that is not covered by that article that meaningful link text helps search engines a lot.

Reply:
Good point 👍